source/server/spongineer/properties.rst

```diff
@@ -0,0 +1,66 @@
+| (boolean, default:false)           | ``ChangeBlockEvent`` if there's no listeners for that event or it's sub         |
+|                                    | events).                                                                        |
++------------------------------------+---------------------------------------------------------------------------------+
+| **sponge.debugShouldFire**         | Logs changes to the ``ShouldFire`` flags when event listeners are               |  
+|                                    | registered/unregistered causing one of the boolean flags to be flipped (whether |
+| (boolean, default: false)          | it's going from ``true`` to ``false`` or ``false`` to ``true``).                |
++------------------------------------+---------------------------------------------------------------------------------+
+| **timings.bypassMax**              | Allows for bypassing the max number of frames to send to the Aikar's timing     |   
+|                                    | API. Setting this will not permit bypassing the limit unless the API is         |  
+| (any value that exists)            | configured to permit it.                                                        |
++------------------------------------+---------------------------------------------------------------------------------+
+|                                    | Adds a StackTrace to each ``StackFrame`` for when either the Sponge             |
+| **sponge.debugcauseframes**        | implementation or a plugin potentially corrupts the ``CauseStackManager`` 's    |
+|                                    | ``StackFrame`` s, whether it's a frame being popped out of order, or popping    |
+| (boolean, default: false)          | an already popped frame. This is useful to find out where the frame was         | 
+|                                    | created.                                                                        |
++------------------------------------+---------------------------------------------------------------------------------+
+|                                    | Similar to the ``StackFrame`` exception creation, this tracks the stack trace   |
+|                                    | for each ``BlockTransaction`` that is captured for a block being changed, a     | 
+| **sponge.debugProxyChanges**       | neighbor notification request, or a ``TileEntity`` change. Useful to find out   |
+|                                    | where each transaction is created, potentially adding more information if there |
+| (boolean, default: false)          | is an exception thrown during the processing of said transactions. Should       |
+|                                    | always be included for logs when reporting issues involving these sort of       |
+|                                    | transactions.                                                                   |
```

Comment:
Likewise, has some performance cost in potentially intensive environments due to the exception creation for every potential `Entity`/`TileEntity`/`Block` being ticked in the game.

Reply:
I chose a warning block over additional bold-face text in the table. Let me know if adding to the table was the original intent.